cmd/cloudx/proxy/cmd_proxy.go

```diff
@@ -289,6 +294,13 @@ func printDeprecations(cmd *cobra.Command, target string) {
 		}
 		sort.Strings(values)
 
-		_, _ = fmt.Fprintf(cmd.ErrOrStderr(), "Attention! We found multiple sources for the project slug. Please clean up environment variables and flags to ensure that the correct value is being used. Found values:\n\n\t%s\n\nOrder of precedence is: %s > %s > %s > --%s\nDecided to use value: %s\n\n", strings.Join(values, "\n\t"), envVarSlug, envVarSDK, envVarKratos, ProjectFlag, target)
+		if flagx.MustGetBool(cmd, LegacyEndpointConfig) {
```

Comment:
If we introduce this parameter, we will have to keep it. Naming it *Legacy* implies that it will be kept for backwards compatibility until a certain date / release version. I think we should decide whether

1. to remove the parameter (because we will break BC anyways at some point)
2. keep the parameter indefinitely supported and rename it to `--allow-multiple-config-source` (name needs improvement)

I think we should just break bc here. The added benefit of this functionality is marginal, and adding more and more config options will in my experience make the software harder to understand and use.

Reply:
I'm all in favour of removing stuff that isn't needed and thus reducing the possibility to introduce bugs.
My thinking behind this option was to provide a workaround for cases, where it's not so easy to "clean up environment variables". Probably this case is very rare and not worth the tradeoff to having to worry about the code in the future.

@aeneasr please let me know if this maybe changes your mind. Otherwise I will go with your proposed option 1 and remove everything :-)

Reply:
Let's go with 1 and if people complain introduce the flag :)